distribution/lib/Standard/Table/0.0.0-dev/src/Data/Expression.enso

```diff
@@ -0,0 +1,31 @@
+from Standard.Base import all
+
+polyglot java import org.enso.table.expressions.ExpressionVisitorImpl
+polyglot java import java.lang.IllegalArgumentException
+polyglot java import java.lang.UnsupportedOperationException
+
+import project.Data.Column.Column
+
+type Expression
+    For_Column expression:Text
+
+    ## PRIVATE
+    evaluate : (Text -> Any) -> (Any -> Any) -> Text -> Text -> Vector Text -> Any
+    evaluate self getColumn makeConstant moduleName="Standard.Table.Data.Column" typeName="Column" var_args_functions=Column.var_args_functions =
```

Comment:
Why is there a default for `moduleName` and `typeName`? It seems to me that every usage site will have it different so I'm not sure if there is a reason to 'favour' the in-memory table by defaulting it. IMO it would be clearer to write the arguments explicitly at each call site.

```suggestion
    evaluate self get_column make_constant module_name type_name var_args_functions=Column.var_args_functions =
```

Also I think we should use Enso snake_case for arguments of such functions. I know they are passed to Java so I'd be fine with using camelCase for internal variables just before/after a Java call. But these are a part of a semi-internal API used in various places and I'm not sure why we'd expose the camelCase here. Especialy as hypothetically we could change the underlying implementation to use sth else than Java and then these names would be outdated - the names are leaking internal implementation details.